.pre-commit-config.yaml

```diff
@@ -1,13 +1,13 @@
 repos:
   - repo: https://github.com/mineiros-io/pre-commit-hooks
-    rev: v0.3.1
+    rev: v0.4.1
     hooks:
+      - id: phony-targets
+      - id: terradoc-validate
+      - id: terradoc-fmt
+      - id: terradoc-generate
+      - id: terramate-generate
       - id: terraform-fmt
       - id: terraform-validate
         exclude: ^examples|.terraform/
-      - id: tflint
-      - id: golangci-lint
-      - id: phony-targets
-      - id: markdown-link-check
-        args: ['-p']   # When adding the -p flag, markdown-link-check will always with an exit code 0, even if dead links are found
-        verbose: true  # Forces the output of the hook to be printed even when the hook passes.
+      - id: tflint
```

Comment:
@SpecialistSun please add new lines at the end of each file. You can configure your IDE to add them automatically to make it easier.

Reply:
@zied-elouaer just curious about adding new lines. Since it seems something rather easy to forget and difficult to know, could we use pre-commit to manage it? At least that is what I have done before to avoid having to configure particular settings.

Reply:
@switchdk that sounds like a good idea. Maybe you should do it :smile: